portal-api/src/v1/gef/controllers/facilities.controller.js

```diff
@@ -86,9 +87,12 @@ const getAllFacilitiesByDealId = async (dealId) => {
 };
 exports.getAllFacilitiesByDealId = getAllFacilitiesByDealId;
 
+exports.isPortalFacilityAmendmentsFeatureFlagEnabled = isPortalFacilityAmendmentsFeatureFlagEnabled;
```

Comment:
```suggestion
exports.isPortalFacilityAmendmentsFeatureFlagEnabled = isPortalFacilityAmendmentsFeatureFlagEnabled;
```

Do we need this line, can we not simply just use `isPortalFacilityAmendmentsFeatureFlagEnabled`?

Reply:
@abhi-markan I need to keep this as it is, because I need to mock the result in unit test